lightyear/src/tests/host_server_stepper.rs

```diff
@@ -247,11 +247,10 @@ impl HostServerStepper {
         self.client_app
             .world_mut()
             .run_system_once(|mut commands: Commands| commands.connect_client());
-
         // Advance the world to let the connection process complete
         for _ in 0..100 {
             if self
-                .client_app
+                .server_app
```

Comment:
this should be `self.client_app`, because the external client needs to check if they're synced

Reply:
yep, originally didnt want to push this change, just overlooked it. fixed now :)